Assignments/load_baseball_data.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+pwd
+docker-compose up mariadb
+
+
+#Got some help from Will but wasnt able to figure it out.
+if ! mysql -h mariadb -uroot -psecret -e 'use baseball'; then
+  mysql -h mariadb -uroot -psecret -e "create database baseball;"
```

Comment:
So I think by executing the mysql directly like this you're going to end up running it on your local machine's install of Mariadb, not the container. I think you'd need to put docker container exec -I mariadb in front of the mysql part so that it accesses the mysql CLI that comes on the Mariadb container